packages/manager/cypress/e2e/core/cloudpulse/linode-widget-verification.spec.ts

```diff
@@ -191,6 +190,16 @@ describe('Integration Tests for Linode Dashboard ', () => {
       .click();
 
     cy.findByText(resource).should('be.visible');
+
+    // Expand the applied filters section
+    ui.button.findByTitle('Filters').should('be.visible').click();
+
+    // Verify that the applied filters
+    cy.get('[data-testid="applied-filter"]').within(() => {
+      cy.get(`[data-qa-value="Region US, Chicago, IL"]`);
```

Comment:
same here as well

Reply:
I have addressed all the comments. Please review them.